src/gadgets/mpt_update.rs

```diff
@@ -1796,19 +1796,14 @@ fn configure_empty_storage<F: FieldExt>(
             .path_type
             .current_matches(&[PathType::Start, PathType::Common]),
     );
+    cb.assert_equal(
+        "hash doesn't change for empty account",
+        config.old_hash.current(),
+        config.new_hash.current(),
+    );
 
     let is_final_segment = config.segment_type.next_matches(&[SegmentType::Start]);
     cb.condition(is_final_segment, |cb| {
-        cb.assert_equal(
-            "old_hash = new_hash",
-            config.old_hash.current(),
-            config.new_hash.current(),
-        );
-        cb.assert_equal(
```

Comment:
Deleted this because we check that `old_value` and `new_value` are 0 in lines 1786 and 1790.